src/language_features/hover.rs

```diff
@@ -1,3 +1,5 @@
+use std::fs;
+
```

Comment:
Separating stdlib from other imports with a blank line seems unconventional for this project. Should wed do that?